### ISSUE TITLE
Allow RPC definition without persistence

### ISSUE THREAD

**Issue by longquanzheng** [indeedeng/iwf-java-sdk#214]
Some RPC like publishing to internal channel doesn't need to read/write any persistence. 
It would be nicer for the code to not have the persistence in the signature. 

Moreover, the RPC request can then tell server to not try load any data/search attributes which will save the perf. 

See server implementation: https://github.com/indeedeng/iwf/blob/main/service/api/service.go#L482 it will skip the Query and use Describe instead. 

Basically the request and set `NONE` as persistenceLoadingType for both data and search attributes.
